backend/service/k8s/deployment_test.go

```diff
@@ -364,3 +365,78 @@ func TestProtoForDeploymentStatus(t *testing.T) {
 		})
 	}
 }
+
+func TestProtoForDeploymentSpecWithProbes(t *testing.T) {
+	t.Parallel()
+	var terminationVar int64 = 30
+	var deploymentTestCases = []struct {
+		id         string
+		deployment *appsv1.Deployment
+	}{
+		{
+			id: "foo",
```

Comment:
when were creating more tests lets use names to reflect what were testing, in this case `LivenessProbe / HTTPGetAction` for example.

Reply:
Gotcha update the name of each of test cases